src/App.test.js

```diff
@@ -1,8 +1,18 @@
-import { render, screen } from '@testing-library/react';
+import { fireEvent, render, screen } from '@testing-library/react';
 import App from './App';
 
-test('renders learn react link', () => {
-  // render(<App />);
-  // const linkElement = screen.getByText(/learn react/i);
-  // expect(linkElement).toBeInTheDocument();
+test('when user clicks on close zoo or open zoo button, text on screen changes', () => {
+  render(<App />);
+  const closeZooButton = screen.queryByText(/Close zoo/i);
+  const openZooButton = screen.queryByText(/Open zoo/i);
+
+  fireEvent.click(closeZooButton);
+
+  const closedZooEl = screen.queryByText('I am never going to financially recover from this');
+  expect(closedZooEl).toBeInTheDocument();
+
+  fireEvent.click(openZooButton);
+
+  const openZooEl = screen.queryByText('My name is Joe Exotic and we are open for business');
+  expect(openZooEl).toBeInTheDocument();
```

Comment:
This is one very good thorough test, but it's only testing 1 component. I'd also like to see a test for maybe adding a cat to the parade as a second test.